src/Lecture1.hs

```diff
@@ -123,4 +136,16 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+
+lowerAndGreater :: Int -> [Int] -> String
+lowerAndGreater n list = showResult (go (0, 0) list)
+    where
+        go :: (Int, Int) -> [Int] -> (Int, Int)
```

Comment:
You don't need to have a pair here, you can just pass two ints separately 🙂 
It will remove the need to use `fst` and `snd` in the implementation
```suggestion
        go :: Int -> Int -> [Int] -> (Int, Int)
```

Reply:
Yeah, I know, actually wanted to make some kinda ADT, but ended with tuple in end... 🤷‍♂️🙂